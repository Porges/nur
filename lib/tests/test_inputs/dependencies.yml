version: "1.0"

default: 
  dependencies: [world]

world:
  dependencies: [hello]
  run: ["echo world"]

hello:
  run: ["echo hello!"]
