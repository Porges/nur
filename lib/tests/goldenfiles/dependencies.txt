--- stdout ---
  hello ┼ — Started task ‘hello’
        │ hello!
        │ — Task ‘hello’ completed
  world ┼ — Started task ‘world’
        │ world
        │ — Task ‘world’ completed
default ┼ — Started task ‘default’
        │ — Task ‘default’ completed
--- stderr ---
